docs/metrics.md

```diff
@@ -0,0 +1,58 @@
+# Metrics provided by Secrets Store CSI Driver
+
+This project uses [opentelemetry](https://opentelemetry.io/) for reporting metrics. This project is under [active development](https://github.com/open-telemetry/opentelemetry-go#release-schedule)
```

Comment:
We might need to update this comment as the project is now in pre-GA phase: https://github.com/open-telemetry/opentelemetry-go#project-status

Reply:
Replaced with above link.